### ISSUE TITLE
Print styles

### ISSUE THREAD

**Issue by mxstbr** [thysultan/stylis.js#44]
## Input

```javascript
injectGlobal`
  @page {
    size: A4 landscape;
  }
  @page :left {
    margin-left: 3cm;
  }
  @media (max-width: 700px) {
    html {
      background: palevioletred;
    }
  }
`
```

## Expected Behavior
style tag in server side rendered head with
```css
@page{size: A4 landscape;}
@page :left{margin-left: 3cm;}
@media (max-width: 700px){html{background: palevioletred;}}
```

## Actual Behavior
style tag in server side rendered head with
```css
@page{{size: A4 landscape;}}
@page :left{{margin-left: 3cm;}}
@media (max-width: 700px){html{background: palevioletred;}}
```

----

Copied from https://github.com/styled-components/styled-components/issues/957

**Comment by TylerWardle** [comment]
seems to be caused by styled-components/src/utils/flatten.js
`export const objToCss = (obj: Object, prevKey?: string): string => {
  const css = Object.keys(obj).map(key => {
    if (isPlainObject(obj[key])) return objToCss(obj[key], key)
    return `${hyphenate(key)}: ${obj[key]};`
  }).join(' ')
  return prevKey ? `${prevKey} {
  ${css}
}` : css
}`

**Comment by TylerWardle** [comment]
Seems to be caused by the objectToCss function of styled-components/src/utils/flatten.js

**Comment by thysultan** [comment]
I think the problem is with stylis, i assumed at the time when i added this that @<page> was a nesting block level selector like `@media` but it's more along the lines of `@keyframes`. Will add to the up coming release.

**Comment by thysultan** [comment]
Patched in 3.2.0
